<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserProfileEditDialog</class>
 <widget class="QDialog" name="UserProfileEditDialog">
  <property name="windowTitle">
   <string>Редактирование профиля</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QDialog {
        background-color: #ffffff;
        font-family: 'Segoe UI', sans-serif;
        color: #222;
    }

    QLabel {
        font-size: 14px;
    }

    QLineEdit, QLineEdit:focus {
        border: 1px solid #ccc;
        border-radius: 4px;
        padding: 4px;
        background-color: #fdfdfd;
    }

    QPushButton {
        padding: 6px 12px;
        border: 1px solid #0078D7;
        background-color: #0078D7;
        color: white;
        border-radius: 4px;
    }

    QPushButton:hover {
        background-color: #005A9E;
    }
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Основная информация</string>
     </property>
     <layout class="QFormLayout" name="formLayout1">
      <item row="0" column="0">
       <widget class="QLabel" name="labelLastName">
        <property name="text">
         <string>Фамилия:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lastNameEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelFirstName">
        <property name="text">
         <string>Имя:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="firstNameEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelMiddleName">
        <property name="text">
         <string>Отчество:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="middleNameEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelEmail">
        <property name="text">
         <string>Почта:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="emailEdit"/>
      </item>
     </layout>
    </widget>
   </item>

   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Изменение пароля</string>
     </property>
     <layout class="QFormLayout" name="formLayout2">
      <item row="0" column="0">
       <widget class="QLabel" name="labelCurrentPassword">
        <property name="text">
         <string>Текущий пароль:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="currentPasswordEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelNewPassword">
        <property name="text">
         <string>Новый пароль:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="newPasswordEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelConfirmPassword">
        <property name="text">
         <string>Подтверждение:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="confirmPasswordEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>

   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
